runners/direct-java/pom.xml

```diff
@@ -361,6 +361,12 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-
+    
+    <!-- needed for eclipse-jdt generated core as the test-jar references classes from this -->
```

Comment:
This is occurring if I run "mvn -Peclipse-jdt clean install" from the root of beam.  This isn't from within Eclipse.   Thus, I'm not sure why the group isn't being picked up.   I'll run with -X and dig through the logs and see if anything pops up as interesting.